media/css/foundation/annual-report.scss

```
@@ -332,3 +347,22 @@
         overflow-x: visible;
     }
 }
+
+.mzp-l-card-half.is-reverse {
+    .mzp-c-card {
+        @media #{$mq-sm} {
+            float: right;
+            margin-right: 0;
+            margin-left: $spacing-xl;
```

I wish I had some docs to point you to but I couldn't find any 🤦‍♀️  We usually use the `bidi` mixin for RTL stuff:

```suggestion
        @include bidi((
            (float, right, left),
            (margin-right, 0, $spacing-xl),
            (margin-left, $spacing-xl, 0),
        ));
```

---

bidi docs! https://protocol.mozilla.org/docs/contributing.html#bidi